source/_docs/eu-region.md

```diff
@@ -1,72 +1,135 @@
+### Verify Domains Route Correctly
+Expose the `x-served-by` response header or grep `AMS` to verify whether the Amsterdam origin shield was used as expected (replace `example.com`):
 
-Allow one week between submitting a ticket until go live. Opening a ticket will not be necessary after early access.
+```bash
+curl -Is https://example.com | grep x-served-by
+```
 
-1.  Add domains to all environments.
-2.  Update DNS values to those provided by the Dashboard. Note that during early access, it may take up to one week for your domain to route directly to the EU region.
-3.  [Open a Support ticket](https://dashboard.pantheon.io/#support){.external} with the following template:
+```bash
+curl -Is https://example.com | grep AMS
+```
 
-    ```nohighlight
-    We are preparing to go live for the site named <example>. Please add the following domains to your next EU deployment:
+The output should look something like:
 
-    example.com
-    www.example.com
-    blog.example.com
-    test.example.com
-    dev.example.com
+```bash
+curl -Is https://dev-rachel-whitton-eu2.pantheonsite.io | grep x-served-by
+x-served-by: cache-ams21041-AMS, cache-jfk8127-JFK
+```
 
-    We understand that domains will be routed to the EU every Tuesday, for all requests submitted by the preceding Friday, and that we will receive confirmation when the update is complete, at which time we can update DNS.
-    ```
-4. After receiving confirmation, celebrate. Your site is running in the EU!
+Time to celebrate. Your site is running in the EU!
 
 ## Coming Soon
 
-More EU Region features are in active development. [Contact us](https://pantheon.io/contact-us){.external} to learn more, and check this guide for updates.
+More International Region features are in active development. [Contact us](https://pantheon.io/contact-us){.external} to learn more, and check this guide for updates.
 
 Coming soon:
 
-  - Dashboard support
-  - Automated domain routing
+  - Disaster Recovery for EU Region
+  - Region Facet on Organization Dashboard
+  - More Regions!
+    - [Fill out this survey](https://www.getfeedback.com/r/hkR9uTAJ){.external} to tell us about your needs.
+
+## Frequently Asked Questions
+### Can I use the EU region for an existing site?
+Yes, however you must migrate your existing site to a new Site Dashboard that was configured for the EU region during creation ([as described above](#create-a-new-site)).
+
+### When will EU Region be available for sites paid by credit card?
+General Availability is planned for late 2019. Tell us more about your needs by [filling out this survey](https://www.getfeedback.com/r/hkR9uTAJ){.external}.
+
+### When will site creation through the Dashboard be available?
+General availability is planned for late 2019.
+
+Contact your account owner or our [Sales team](https://pantheon.io/contact-us){.external} to learn about Pantheon's migration services or review the [relaunch procedure](/docs/relaunch/) for steps on how to migrate the site yourself.
```

Comment:
Consider referencing the relaunch doc earlier where we're telling people to do a manual migration for existing sites.